.github/workflows/integration-test.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           echo "CEPH_USERNAME=admin" >> $GITHUB_ENV
           echo "CEPH_POOLNAME=devpool" >> $GITHUB_ENV
           echo "CEPH_CLIENTNAME=client.admin" >> $GITHUB_ENV
+          echo "CEPH_DISK_SIZE=65260281856" >> $GITHUB_ENV
```

Comment:
Can we somehow dynamically calculate this value?

Reply:
Will do so in a separate PR, captured in an issue.